backend/pkg/lobby/store.go

```diff
@@ -18,17 +18,19 @@ func NewStore(db *sqlx.DB) core.LobbyStore {
 }
 
 func (s *lobbyStore) List(ctx context.Context) ([]*core.Lobby, error) {
-	rows, err := s.db.QueryxContext(ctx, `SELECT 
-       id, restaurant, owner, expires FROM lobbies`)
+	rows, err := s.db.QueryxContext(ctx, `SELECT l.id, l.restaurant, r.name, l.owner,
+ 								l.expires, l.geolat, l.geolon FROM lobbies l
+ 								JOIN restaurants r ON r.id = l.restaurant`)
```

Comment:
I'm not sure if we shouldn't use LEFT JOIN here. What do you think @kirinson321 ?